src/__tests__/__snapshots__/index.js.snap

```diff
@@ -116,7 +122,7 @@ exports[`merges composed component styles for reasonable overrides 1`] = `
 }
 
 <div
-  class="other classes are not removed css-1k8eh41"
+  class="other classes are not removed css-1k8eh41 glamorous_glamorous_glamorous_div__"
```

Comment:
Yeah, I'm not sure why the rules are the way they are, but yeah, you can only have certain characters in the class name of an element.

I think we can discuss things more in another issue if we want to change the default display name of glamorous components.